iocore/cache/Cache.cc

```diff
@@ -2193,6 +2193,11 @@ CacheVC::handleReadDone(int event, Event *e)
       }
       goto Ldone;
     }
+    if (DISK_BAD(vol->disk)) {
+      io.aio_result = -1;
+      Warning("Canceling cache read: disk is bad.");
```

Comment:
Can we change this to use `vol->hash_text.get()` to indicate which disk is bad?
`Warning("Canceling cache read: disk %s is bad.", vol->hash_text.get());`